FusionAuth/fusionauth-site
# Algolia search for docs

This PR replaces our current doc search with one that uses algolia. This means that every time we push a build to production, we'll update the algolia instance (it does checksums so only pushes new data). I have already done that once, so the search should work just fine, as there is a readonly API key checked in.

To test:
* check out `add-algolia` in `fusionauth-style`
* run `sb int`
* check out `algolia` in `fusionauth-site`
* run `sb css` to pull down the latest css file.
* bundle install #pick up new gems
* set env vars as documented in the readme. I'll post them in slack.
* restart the web server if needed (so it picks up config values)
* visit http://localhost:4000/docs/v1/tech/ 
* enter search terms in the new search box

## Release plan

* Sign up for paid algolia account (lowest paid tier should be fine)
* change back to individual account (from dev) add brian and daniel
* make sure that the envt vars for the keys are available
* remind folks doing website releases they'll need to set the vars in their config (and update the docs if needed)
* Merge and release the CSS
* do a release
